uk.co.compucorp.civicrm.hrcontactactionsmenu/CRM/HRContactActionsMenu/Helper/UserInformationActionGroup.php

```diff
@@ -80,8 +80,7 @@ private function getMenuButton($params) {
     $button = new ActionsGroupButtonItem($params['label']);
     $button->setClass($params['class'])
       ->setIcon($params['icon'])
-      ->setUrl($params['url'])
-      ->addBottomMargin();
```

Comment:
Will all the buttons have bottom margin now by default?

Reply:
@tunbola  yes!